IDEAS/Buildings/Components/BaseClasses/ZoneLwGainDistribution.mo

```diff
@@ -38,19 +38,29 @@ model ZoneLwGainDistribution "distribution of radiative internal gains"
         origin={40,100})));
 
 protected
-  Real[nSurf] areaAbsDifSol=area .* epsSw "shortwave emissivity weighted areas";
-  Real areaAbsDifTotSol=sum(areaAbsDifSol)
-    "sum of shortwave emissivity weighted areas";
-  Real[nSurf] areaAbsGain=area .* epsLw "longwave emissivity weighted areas";
-  Real areaAbsTotGain=sum(areaAbsGain)
+  parameter Real[nSurf] areaAbsDifSol = zeros(nSurf)
```

Comment:
does this work? since you already assign a parameter value. I'd expect you need to do it like this:

parameter Real[nSurf] areaAbsDifSol(fixed=false)